src/Api.php

```diff
@@ -34,12 +38,51 @@ public function checkoutSession($rawPayload)
     public function getOrders($params = [])
     {
         $parsedParams = [
-            'page' => isset($params['page']) ? $params['page'] : 1,
-            'count' => isset($params['count']) ? $params['count'] : Smartpay::DEFAULT_PAGE_COUNT,
+            'pageToken' => isset($params['pageToken']) ? $params['pageToken'] : null,
+            'maxResults' => isset($params['maxResults']) ? $params['maxResults'] : null,
+            'expand' => isset($params['expand']) ? $params['expand'] : '',
         ];
 
         return new OrdersResponse(
             $this->client->get('/orders', $parsedParams)
         );
     }
+
+    public function getOrder($params = [])
+    {
+        $id = $params['id'];
+        $parsedParams = [
+            'expand' => isset($params['maxResults']) ? $params['maxResults'] : null,
+        ];
+
+        return new OrdersResponse(
+            $this->client->get("/orders/{$id}", $parsedParams)
+        );
+    }
+
+    public function createPayment($rawPayload)
+    {
+        $request = new PaymentRequest($rawPayload);
+        return new PaymentResponse(
+            $this->client->post('/payments', $request->toRequest())
+        );
+    }
+
+    public function capture($rawPayload)
+    {
+        return $this->createPayment($rawPayload);
```

Comment:
minor suggestions:

I think we only need to define one public function for the same purpose. Maybe we can define the method `createPayment` as a private function?